src/main/java/org/apache/commons/csv/CSVFormat.java

```diff
@@ -288,6 +294,18 @@ public Builder setAllowDuplicateHeaderNames(final boolean allowDuplicateHeaderNa
             return this;
         }
 
+        /**
+         * Sets whether the last field on the last line, if quoted, can have no closing quote when the file ends, {@code true} if this is ok,
+         * {@code false} if {@link IOException} should be thrown.
+         *
+         * @param allowEOFWithoutClosingQuote whether to allow the last field on the last line to have a missing closing quote when the file ends,
+         *                                    {@code true} if so, or {@code false} to cause an {@link IOException} to be thrown.
+         */

```

Comment:
Add Javadoc since tags to new public and protected elements.

Reply:
Done, used 1.10.0.